.docker/build/php/php.ini

```diff
@@ -0,0 +1,19 @@
+[PHP]
+
+;;;;;;;;;;;;;;;;;;;
+; About php.ini   ;
+;;;;;;;;;;;;;;;;;;;
+; PHP's initialization file, generally called php.ini, is responsible for
+; configuring many of the aspects of PHP's behavior.
+
+date.timezone = 'Europe/Paris';
+short_open_tag = Off;
+magic_quotes_gpc = Off;
+register_globals = Off;
+session.auto_start = Off;
+error_reporting = E_ALL;
+log_errors = On;
+display_errors = On;
+error_log = /dev/stderr;
+memory_limit = -1;
```

Comment:
To fix : 128M

Reply:
Fixed in 27d41f822f32d2b15ce1dfd1f214c1e81c5ee876